source/examples/rapids-azureml-hpo/notebook.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>Could you add a paragraph or two here with a small introduction and an explanation of what this notebook does?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>